test/utils/image/manifest.go

```diff
@@ -226,7 +223,7 @@ func initImageConfigs() map[int]Config {
 	configs[JessieDnsutils] = Config{e2eRegistry, "jessie-dnsutils", "1.0"}
 	configs[Kitten] = Config{e2eRegistry, "kitten", "1.0"}
 	configs[Nautilus] = Config{e2eRegistry, "nautilus", "1.0"}
-	configs[NFSProvisioner] = Config{quayIncubator, "nfs-provisioner", "v2.2.2"}
+	configs[NFSProvisioner] = Config{k8sCSI, "nfs-provisioner", "v2.2.2"}
```

Comment:
nfs-provisioner is technically not part of the kubernetes-csi project, but we can put it here temporarily just for unblocking pull jobs. As part of moving the nfs provisioners to kubernetes-sigs, we'll also need to revamp the build and release process for them. cc @kmova

Reply:
Agreed. thanks @msau42